tgui/packages/tgui/interfaces/UploadPanel.js

```diff
@@ -0,0 +1,53 @@
+import { useBackend } from "../backend";
+import { Button, ProgressBar, LabeledList, Box, Section } from "../components";
+import { Window } from "../layouts";
+
+export const UploadPanel = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { selected_target, new_law, id, transmitting, hacked } = data;
+  return (
+    <Window>
+      <Window.Content>
+        <Section title="Silicon Law Upload">
+          <LabeledList>
+            <LabeledList.Item
+              label="Selected Target">
```

Comment:
Это оно само так отформатировало? Лучше или в одну строку, или `>` на новой.
```suggestion
            <LabeledList.Item label="Selected Target">
```